internal/database/database.go

```diff
@@ -25,17 +26,19 @@ import (
 	"github.com/eiffel-community/eiffel-goer/internal/database/drivers/mongodb"
 )
 
-// Get a new Database.
-func Get(connectionString string, logger *log.Entry) (drivers.DatabaseDriver, error) {
-	var drivers = []drivers.DatabaseDriver{&mongodb.Driver{}}
+// Slice of drivers that are supported at the moment.
+// Variable is exported to assist with testing of this and other packages.
```

Comment:
Nit: Comments should start with the name of the identifier (i.e. "Drivers contains the drivers that are supported...").

Reply:
Grr. I agree.